controller/controller_server/csi_controller_server.py

```diff
@@ -706,9 +706,7 @@ def _get_object_final_name(self, request, max_name_prefix_length, max_name_lengt
         return full_name[:max_name_length]
 
     def _join_object_prefix_with_name(self, prefix, name):
```

Comment:
since the function is now a one liner - do we still need it?

Reply:
still relevance?

Reply:
no